rg_instructor_analytics/views/Suggestion.py

```diff
@@ -117,13 +117,15 @@ def get_percent(total, put):
             return .0 if not (total and put) else float(put) / float(total)
 
         subsections_percent = np.array([get_percent(unit['student_count_in'], unit['student_count']) for unit in units])
+        # Get an array of elements, that represents a condition that passed to index operator.
```

Comment:
s/Get an array of elements, that represents a condition that passed to index operator./Get an array of elements, which are satisfied to the condition in the brackets. (numpy's feature)